src/main/java/net/mcreator/ui/help/HelpLoader.java

```diff
@@ -88,19 +88,19 @@ public static String loadHelpFor(IHelpContext helpContext) {
 				if (getFromCache(helpContext.getEntry()) != null) {
 					helpString.append(renderer.render(parser.parse(getFromCache(helpContext.getEntry()))));
 				} else {
-					helpString.append("No help entry is defined for <tt>").append(helpContext.getEntry())

```

Comment:
"yet" word is not translated. I suggest you use the token replacements for this (`{0}, ...`).